Hazel/src/Platform/Windows/WindowsWindow.h

```diff
@@ -3,6 +3,7 @@
 #include "Hazel/Window.h"
 #include "Hazel/Renderer/GraphicsContext.h"
 
+#include <glad/glad.h> // required in WindowsWindow.cpp; but must be included before GLFW
```

Comment:
Best way for me is to remove glad and glfw includes in header file and forward declare glfw objects.

Reply:
Since it is only moving it from the cpp to the header file... it's easier to move that single include around rather than forward declare all glfw objects every time we add new code...